### PR TITLE
INTMDB-630: [CFN - PI] Update backup schedule to move the read-only property from definitions to the properties path

### PR DESCRIPTION
**Pull request by andreaangiolillo**
## Description

Ticket: https://jira.mongodb.org/browse/INTMDB-630

We should update backup schedule to move the read-only property from definitions to the properties path.
See [482](https://github.com/cdklabs/cdk-import/issues/482)

Link to any related issue(s): 

## Type of change:

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Required Checklist:

- [x] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added any necessary documentation (if appropriate)
- [x] I have run `make fmt` and formatted my code
- [x] For CFN Resources: I have released by changes in the private registry and proved by change works in Atlas

## Further comments

I would recommend enabling **hide whitespace** to review this PR. 
Screenshot 2023-02-23 at 13 47 41

### CODE REVIEW

File: cfn-resources/cloud-backup-schedule/mongodb-atlas-cloudbackupschedule.json [hunk 1]
```diff
@@ -1,279 +1,263 @@
...
+    "/properties/UpdateSnapshots",
+    "/properties/DeleteCopiedBackups"
+  ],
+  "readOnlyProperties": [
+    "/properties/ClusterId",
+    "/properties/NextSnapshot",
+    "/properties/Policies/*/PolicyItems/*/ID",
+    "/properties/ProjectId",
+    "/properties/ClusterName",
+    "/properties/Links"
```
Comment (andreaangiolillo) [inline cfn-resources/cloud-backup-schedule/mongodb-atlas-cloudbackupschedule.json:231]: This is the main change

**Review by govardhanpagidi — APPROVED** [review]

**Review by colm-quinn — APPROVED** [review]
LGTM

File: cfn-resources/cloud-backup-schedule/mongodb-atlas-cloudbackupschedule.json (hunk 1 above)
Comment (SuperMohit) [reply cfn-resources/cloud-backup-schedule/mongodb-atlas-cloudbackupschedule.json:231]: @andreaangiolillo ProjectId would be input by the user. For now manually fixing it on cdk side.

Comment (andreaangiolillo) [reply cfn-resources/cloud-backup-schedule/mongodb-atlas-cloudbackupschedule.json:231]: fixing it here https://github.com/mongodb/mongodbatlas-cloudformation-resources/pull/414

**Pull request merged**